advice-dont-ask-about-solutions.md

```diff
@@ -0,0 +1,56 @@
+# Don't ask about solutions[^1]
+
+You may have been sent this document because **you asked about solutions** to a problem,
+rather than the problem itself, or without explaining your original problem.
+This is called ["The XY Problem"](https://en.wikipedia.org/wiki/XY_problem)
+
+We often encounter problems that appear to have multiple, possible solutions. These
+solutions themselves however could cause more problems, which in turn have multiple
+solutions, which in turn may cause more problems...
+
+What may have happened here is that you just asked a question that mostly consists
+of what you tried or waht you're trying to do, rather than giving context on **why
+it is you think you have to do this**. For example,
```

Comment:
```suggestion
What may have happened here is that you asked a question that mostly consists
of what you tried or what you're trying to do, rather than giving context on **why
it is you think you have to do this in the first place**. For example,
```